lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_xla_type.cpp

```diff
@@ -848,6 +844,16 @@ at::Tensor select(const at::Tensor& self, int64_t dim, int64_t index) {
       LazyTensor::select(bridge::GetLtcTensor(self), dim, index));
 }
 
+at::Tensor slice(const at::Tensor& self, int64_t dim,
+                 c10::optional<int64_t> start, c10::optional<int64_t> end,
+                 int64_t step) {
+  int64_t start_val = start.has_value() ? start.value() : 0;
+  int64_t end_val = end.has_value() ? end.value() : INT64_MAX;
```

Comment:
are there any considerations here- i guess MAX is the safest thing we can do and be flexible for dynamic shapes?

Reply:
It's not exclusively about dynamic shapes, it just makes the representation nicer without having to handle all 4 combinations of optionals.